api/client/src/main/java/org/projectnessie/client/http/impl/jdk11/JavaRequest.java

```diff
@@ -162,6 +163,10 @@ public org.projectnessie.client.http.HttpResponse executeRequest(Method method,
           // ignore
         }
       }
+      HttpAuthentication auth = this.auth;
```

Comment:
Looks like this piece should live in `BaseHttpRequest` (shared w/ other implementations)